x-pack/legacy/plugins/apm/public/components/app/APMIndicesPermission/index.tsx

```diff
@@ -96,12 +111,30 @@ const PermissionPage = ({ onEscapeHatchClick }: Props) => {
                 </h2>
               }
               body={
-                <p>
-                  {i18n.translate('xpack.apm.permission.description', {
-                    defaultMessage:
-                      "We've detected your current role in Kibana does not grant you access to the APM data. Please check with your Kibana administrator to get the proper privileges granted in order to start using APM."
-                  })}
-                </p>
+                <>
+                  <p>
+                    {i18n.translate('xpack.apm.permission.description', {
+                      defaultMessage:
+                        'We’ve detected your current role in Kibana does not grant you access to the APM data. Please check with your Kibana administrator to get the proper privileges granted to start using APM.'
```

Comment:
A couple of changes to the copy:
 - A user can have multiple roles so I don't think we should talk about a specific role.
 - Instead of saying "APM data" I think it's cleare to say say "APM indices"
 - I think we should point out that this doesn't need to be critical (they can still use the APM app) but some data might be missing


```suggestion
                        'Your user doesn't have access to all APM indices. You can still use the APM app but some data will be missing. To fix this warning you must be granted access to the following indices:'
```

Reply:
> To fix this warning you must be granted access to the following indices:

I think this is sufficient instead;

> You must be granted access to the following indices:

Otherwise better 👍

Reply:
> You must be granted access to the following indices:

I think this is fine but just to be clear: an admin might deliberately limit certain users access to specific indices (eg. the error indices) because they only should look at trace data. The text "You must be granted access to the following indices" might make those users think it's a problem that they don't have access to the error indices, and therefore contact their admin.